src/services/conversationsService.js

```diff
@@ -326,6 +326,42 @@ const setSIPEnabled = async function(token, newState) {
 	})
 }
 
+/**
+ * Verify SIP DialIn pin
+ *
+ * @param {string} token The token of the conversation
+ * @param {string} pin The pin to verify
+ */
+const verifyDialInPin = async function(token, pin) {
+	return axios.post(generateOcsUrl('apps/spreed/api/v4/room/{token}/verify-dialin', { token }), { pin })
+}
+
+/**
+ * Verify SIP DialOut number
+ *
+ * @param {string} token The token of the conversation
+ * @param {string} number The number to verify
+ * @param {object} options additional details
+ */
+const verifyDialOutNumber = async function(token, number, options) {
```

Comment:
this endpoint is for the HPB/SIP bridge and needs a secret header the UI does not know.